### PR TITLE
[# 143081657] As a patient care team member, so I can keep track of all tabs after a timeout, redirect back to previous pages after I sign in.

### PR DESCRIPTION
**Pull request by kristinebrown10**
As a patient care team member, so I can keep track of all tabs after a timeout, redirect back to previous pages after I sign in.

[Tracker #143081657](https://www.pivotaltracker.com/story/show/143081657)

[Jarvis PR](https://github.com/ConsultingMD/jarvis/pull/7377)
[Banyan PR](https://github.com/ConsultingMD/banyan/pull/456)

### CODE REVIEW

File: lib/grnds/sso/configuration.rb [hunk 1]
```diff
@@ -22,9 +22,11 @@ def vpn
     end
 
     class << self
-      def sign_in_url
+      def sign_in_url(query={})
         configuration = Grnds::Sso.configuration
-        configuration.base_site + configuration.sign_in_post_fix
+        url = configuration.base_site + configuration.sign_in_post_fix
+        url += "?#{query.to_param}" if params.present?
```
Comment (suwyn) [inline lib/grnds/sso/configuration.rb]: @kristinebrown10 Should be 

`url += "?#{query.to_param}" if query.present?`

Comment (kristinebrown10) [inline lib/grnds/sso/configuration.rb]: 😶

**Comment by kristinebrown10** [comment]
@suwyn @jstrater  bug fixed

**Comment by jstrater** [comment]
No specs in this repo? 😢

**Pull request merged**